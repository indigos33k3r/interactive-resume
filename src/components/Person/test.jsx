import React from 'react'
import {shallow} from 'enzyme'

describe('Person', () => {
  it('renders <Person />', () => {
    const component = render()

    expect(component).toMatchSnapshot()
  })

  function render() {
    const defaultProps = {
      personClass: 'boy1-briefcase',
      children: 'Some children text'
    }
    const Person = require('.').default

    return shallow(<Person {...defaultProps} />)
  }
})
