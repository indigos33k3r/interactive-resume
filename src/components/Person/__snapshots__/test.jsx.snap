// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Person renders <Person /> 1`] = `
<div
  className="person person--boy1-briefcase"
>
  Some children text
</div>
`;
